editor/package.json

```diff
@@ -196,7 +196,7 @@
     "react-merge-refs": "1.0.0",
     "react-onclickoutside": "6.9.0",
     "react-popper": "2.2.3",
-    "react-router": "^6.15.0",
+    "react-router": "6.14.2",
```

Comment:
too bad we can't have comments here, it'd be important to note that this version needs to match the version used by remix